src/components/UserConsent/__tests__/ReloadContent.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { h } from 'preact'
+import { mount, shallow } from 'enzyme'
+
+import ReloadContent from '../ReloadContent'
+import MockedLocalised from '~jest/MockedLocalised'
+
+const defaultProps = {
+  onPrimaryButtonClick: jest.fn(),
+}
+
+describe('ReloadContent', () => {
+  it('renders without crashing', () => {
+    const wrapper = shallow(
```

Comment:
`shallow` is less reliable than `mount` in unit testing because it doesn't trigger deeper renders than the top-level elements. I always suggest adding `mount` tests for better cases covering.

Reply:
I see. Will refactor to use `mount` instead then.